libmscore/types.h

```diff
@@ -420,7 +420,9 @@ enum class Tid {
       USER4,
       USER5,
       USER6,
-      TEXT_STYLES
+      // special, no-contents, styles used while importing older scores
+      TEXT_STYLES,           // used for user-defined styles
+      IGNORED_STYLES         // used for styles no longer relevant (mainly Figured bass text style)
```

Comment:
Might be for "Repeat Text" too? ISTR your scores using those.

Reply:
Possibly. I am not entirely sure yet that the "RepeatText" does not come from some customisation of mines. And this PR and the issue it refers to are already well defined in title, topic, code, etc. So, maybe another PR when I will be sure it is an MS problem and not mine.

Reply:
You're probably right, it is is separate issue.

Reply:
Hmm, MuseScore 2 had 3 different Repeat text styles (Repeat Text Left, Repeat Text Right and Repeat Text), MuseScore 3 has only 2 (Repeat Text Left, Repeat Text Right), so it is not your fault, other that that you customized that 'excess' Repeat Text.
So ignoring it may indeed be the way to go?

Reply:
If there is nowhere to put it, I believe the way is to ignore it. Yes, I customised it, but eventually it was not really usefully, so I totally forgot about it! 

Maybe it would be better to drop this PR and push a new one ignoring both?

Reply:
I'd just add it to this here "while at it also ignore the no longer existing text style 'Repeat Text'", which as far as I can tell is just one additional line of code.

Reply:
Updated with "Repeat Text" too.